### PR TITLE
feat(route): add 我不是盐神

### PR DESCRIPTION
**Pull request by ruoshui9527**
## Involved Issue / 该 PR 相关 Issue

Close #

## Example for the Proposed Route(s) / 路由地址示例

```routes
/onehu
```

## New RSS Route Checklist / 新 RSS 路由检查表
  
- [ ] New Route / 新的路由
  - [ ] Follows [v2 Script Standard](https://docs.rsshub.app/joinus/advanced/script-standard) / 跟随 [v2 路由规范](https://docs.rsshub.app/zh/joinus/advanced/script-standard)
- [ ] Documentation / 文档说明
- [ ] Full text / 全文获取
  - [ ] Use cache / 使用缓存
- [ ] Anti-bot or rate limit / 反爬/频率限制
  - [ ] If yes, do your code reflect this sign? / 如果有, 是否有对应的措施?
- [ ] [Date and time](https://docs.rsshub.app/joinus/advanced/pub-date) / [日期和时间](https://docs.rsshub.app/zh/joinus/advanced/pub-date)
  - [ ] Parsed / 可以解析
  - [ ] Correct time zone / 时区正确
- [ ] New package added / 添加了新的包
- [ ] `Puppeteer`

## Note / 说明
知乎盐选文章

### CODE REVIEW

**Review by TonyRL — COMMENTED** [review]
- Missing documentation
- https://github.com/DIYgod/RSSHub/actions/runs/7706910101/job/21003218101?pr=14348

File: lib/v2/onehu/maintainer.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+module.exports = {
+    '/': ['9527'],
```
Comment (TonyRL) [inline lib/v2/onehu/maintainer.js:2]:
```suggestion
    '/': ['ruoshui9527'],
```

File: lib/v2/onehu/radar.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+module.exports = {
+    'onehu.xyz': {
+        _name: 'Onehu',
```
Comment (TonyRL) [inline lib/v2/onehu/radar.js:3]: https://github.com/DIYgod/RSSHub/blob/eec19e594c95b89776b171831fda39ffa57f5d31/website/docs/routes/new-media.mdx#L5292
```suggestion
        _name: '我不是盐神',
```

File: lib/v2/onehu/radar.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+        '.': [
+            {
+                title: '我是盐神',
```
Comment (TonyRL) [inline lib/v2/onehu/radar.js:6]: https://github.com/DIYgod/RSSHub/blob/eec19e594c95b89776b171831fda39ffa57f5d31/website/docs/routes/new-media.mdx#L5294
```suggestion
                title: '首页',
```

File: lib/v2/onehu/radar.js [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+                docs: 'https://docs.rsshub.app/routes/new-media#onehu',
```
Comment (TonyRL) [inline lib/v2/onehu/radar.js:7]: https://github.com/DIYgod/RSSHub/blob/eec19e594c95b89776b171831fda39ffa57f5d31/website/docs/routes/new-media.mdx#L5292
```suggestion
                docs: 'https://docs.rsshub.app/routes/new-media#wo-bu-shi-yan-shen',
```

**Comment by ruoshui9527** [comment]
Thank you, have made the suggested changes

**Pull request merged**